consensus/reactor_test.go

```diff
@@ -689,7 +689,7 @@ func capture() {
 // Ensure basic validation of structs is functioning
 
 func TestNewRoundStepMessageValidateBasic(t *testing.T) {
-	testCases := []struct { // nolint: maligned
+	testCases := []struct { //nolint: maligned
```

Comment:
`maligned` appears to be deprecated. It is also a linter aimed at saving space when changing field order would allow structs to be more tightly packed. While nifty, this is such a minimal savings in most cases that I really do not see the value of this linter and think we should completely remove it. Almost any amount of time fixing the errors it gives us will not be worth it in terms of space savings cost.